front/lib/api/assistant/configuration/process.ts

```diff
@@ -0,0 +1,157 @@
+import type {
+  DataSourceConfiguration,
+  ModelId,
+  ProcessConfigurationType,
+} from "@dust-tt/types";
+import _ from "lodash";
+import { Op } from "sequelize";
+
+import { DEFAULT_PROCESS_ACTION_NAME } from "@app/lib/api/assistant/actions/names";
+import { renderRetrievalTimeframeType } from "@app/lib/api/assistant/configuration/helpers";
+import { AgentDataSourceConfiguration } from "@app/lib/models/assistant/actions/data_sources";
+import { AgentProcessConfiguration } from "@app/lib/models/assistant/actions/process";
+import { DataSource } from "@app/lib/models/data_source";
+import { Workspace } from "@app/lib/models/workspace";
+import { DataSourceViewResource } from "@app/lib/resources/data_source_view_resource";
+import { DataSourceViewModel } from "@app/lib/resources/storage/models/data_source_view";
+
+export async function fetchAgentProcessConfigurationsActions({
```

Comment:
This will add latency. Before, we were getting the action config with the rest of action configs, and had a second step for all the "secondary" resources linked to action configs. Now we have an extra step